pdf-over-gui/src/main/java/at/asit/pdfover/gui/utils/UpdateCheckManager.java

```diff
@@ -49,31 +51,27 @@ private static void setStatus(Status status) {
 
     private static String latestVersionNotified = null;
     private static Status runCheck(Shell shell) {
-        HttpClient client = (HttpClient) BKUHelper.getHttpClient();
-        GetMethod method = new GetMethod(Constants.CURRENT_RELEASE_URL);
-        try {
-            client.executeMethod(method);
-            final String version = method.getResponseBodyAsString().trim();
-            if (!VersionComparator.before(Constants.APP_VERSION, version))
-                return Status.UP_TO_DATE;
+        try (final CloseableHttpClient httpClient = HttpClients.custom().useSystemProperties().build()) { /* TODO replace post #37 merge */
```

Comment:
this should use the unified httpclient construction after #37, so this PR is blocked by that PR